source/run-agg-pipelines.txt

```diff
@@ -140,69 +181,45 @@ First Stage
      The ``$project`` pipeline in the example suppresses the ``_id``
      field from the documents it passes to the next stage.
 
+  The ``$project`` pipeline in the example transforms the example
```

Comment:
This is a slight misnomer throughout. `$project` is a **stage** not a pipeline. The pipeline is the overall array which contains individual stages. This needs to be addressed in several other places.

Reply:
updated